pkg/reconcile/pipeline/infinispan/handler/manage/conditions.go

```diff
@@ -54,6 +54,9 @@ func OperatorStatusChecks(i *ispnv1.Infinispan, ctx pipeline.Context) {
 	}
 	// Pod name is changed, means operator restarted
 	if i.Status.Operator.Pod != operatorPod {
+		if i.Spec.Autoscale != nil {
+			ctx.Log().Info("Autoscale is no longer supported. Please remove spec.autoscale field.", "Request.Namespace", i.Namespace, "Request.Name", i.Name)
```

Comment:
Should this be an error so that it has more visibility?

Reply:
should we do as we did for [cacheservice](https://github.com/infinispan/infinispan-operator/commit/5b121f826f4499a4b8bfb3417de360e9e07a54eb), warn + event?

Reply:
Yes, that's a good idea :+1:

Reply:
The current code is still logging an info message. I know the API doesn't expose a `.Warn(..)` method, so we either stick with `info()` and risk the log being missed or go with `.Error(..)` to increase it's exposure.

Reply:
ok for error